__tests__/app.test.js

```diff
@@ -0,0 +1,29 @@
+const app = require("../app");
+const request = require("supertest");
+const testData = require("../db/data/test-data/index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  if (db.end) db.end();
+});
+
+describe("GET /api/categories", () => {
```

Comment:
Looks good for the endpoint but we could also have a separate test and functionality for general 404 error handling at this point too. 😄